packages/ngx-deploy-npm/package.json

```diff
@@ -11,9 +11,9 @@
     "node": ">=14.0.0"
   },
   "peerDependencies": {
-    "@nrwl/devkit": "^14.0.0 || ^15.0.0",
-    "@nrwl/workspace": "^14.0.0 || ^15.0.0",
-    "nx": "^14.0.0 || ^15.0.0"
+    "nx": "^14.0.0 || ^15.0.0 || ^16.0.0",
+    "@nx/devkit": "^14.0.0 || ^15.0.0 || ^16.0.0",
+    "@nx/workspace": "^14.0.0 || ^15.0.0 || ^16.0.0"
```

Comment:
I am not sure this is actually possible... does `@nx` scoped versions of devkit and workspace exist for v14 or v15?

Reply:
> All of the changes make sense to me with the exception of the supported peer dependency versions. Depending on when the @nx scoped packages were introduced this might have to be a major bump breaking change for v16+

Good catch!
According to the Nx team, they will be continuing publishing on the old npm scope after V17. Makes sense to continue using the old npm scope and then create a major version as you suggested.

Reply:
For simplicity, we decided to introduce a breaking change and support only v16